applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -91,5 +96,19 @@ def Initialize(self):
         (self.solver).SetEchoLevel(self.settings["echo_level"].GetInt())
         (self.solver).Initialize()
 
-    def AdvanceInTime(self, current_time):
-        raise Exception("AdvanceInTime is not implemented. Potential Flow simulations are steady state.")
```

Comment:
we don not need it at this point since this is inside the fluid solver and this function is actually not called.
In co_simulation_steady_analysis we do not "AdvanceInTime" and the MainCoSim.py calls this class instead of co_simulation_analysis which itself calls for that function

Reply:
In that case I would leave the function with the exception as it was before.